static/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -99,13 +100,20 @@ export default class InstalledIntegration extends React.Component<Props> {
         ? this.disableConfirmProps
         : this.removeConfirmProps;
 
+    const alertText = getAlertText([integration]);
+
     return (
       <Access access={['org:integrations']}>
         {({hasAccess}) => (
           <IntegrationFlex key={integration.id} className={className}>
             <IntegrationItemBox>
               <IntegrationItem integration={integration} />
             </IntegrationItemBox>
+            {alertText && (
+              <Alert type="warning" icon={<IconFlag size="sm" />}>
+                {alertText}
+              </Alert>
+            )}
```

Comment:
It seems kinda squished, but I don't think that's all that important. nit: i guess.

Reply:
@mgaeta I agree with Leander here. The first screenshot in the description of this PR does look squished. Can we make it look like this: https://user-images.githubusercontent.com/15368179/82481962-f7190d80-9a8a-11ea-8eb4-91eed67926ea.png

Here, the banner is at the top and the upgrade now button is on the configuration it applies to.

(The wording should be: "Update to the latest version of our Slack app to get access to personal and team notifications." instead of what's in the screenshot)